src/booster.py

```diff
@@ -0,0 +1,84 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+import typer
+import yaml
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏Can we use toml instead?

I have a pet peeve about yaml - I find it hard to format correctly for some reason. Toml is a simpler standard, we already use it (`pyproject.toml`) and as an added bonus - reading toml is built into recent versions of Python: https://docs.python.org/3/library/tomllib.html